app/views/chargeback_rate/_cb_rate_show.html.haml

```diff
@@ -49,9 +49,9 @@
         %td{:rowspan => num_tiers}
           = h(rate_detail_group(detail.chargeable_field[:group]))
         %td{:rowspan => num_tiers}
-          = detail.chargeable_field[:description]
+          = _(detail.chargeable_field[:description])
           %br
-          = Dictionary.gettext(detail.chargeable_field.metric_key, :type => :column, :notfound => :titleize)
+          = _(Dictionary.gettext(detail.chargeable_field.metric_key, :type => :column, :notfound => :titleize))
```

Comment:
Some of the strings here are being translated twice because they are also in `manageiq/locale/en.yml`. Should this translation stay here or should the missing strings be added to `en.yml`?

![image](https://user-images.githubusercontent.com/64800041/96778371-9d11d080-13b9-11eb-83bf-61f3c0201725.png)

Reply:
> should the missing strings be added to `en.yml`?

What strings do you see missing?

Reply:
This is a picture from the master branch 
![image](https://user-images.githubusercontent.com/64800041/96878335-b746bf80-1448-11eb-99ad-ab62a971bf8c.png)

Reply:
I don't think the image explained things very well but basically the non-highlighted strings in this image that are marked for translation are coming from `en.yml`. The changes I made above ignore this and just mark everything for translation. My question was, whether the highlighted strings should be added to `en.yml` and the above code changes undone or to just leave my changes as is.

Reply:
There's no need to apply `_()` to `Dictionary.gettext()`, since whatever is returned by theat routine already went through `_()`.

Reply:
The problem is that not all the strings are getting marked for translation 
<img src="https://user-images.githubusercontent.com/64800041/97879986-986ce680-1cee-11eb-8f76-305285c84567.png" width="300">

Reply:
Right. But using `_()` here won't help at all.

Reply:
Which leads to my original question of whether these strings should be added to `en.yml`.

Reply:
It really depends on where those strings come from. Do the come from some a fixture yaml file perhaps?

Reply:
It looks like the strings are coming from `manageiq/product/charts/miq_reports/vm_chargeback.yaml` and `manageiq/db/fixtures/chargeback_rates.yml`.

Reply:
In that case you need to update the following config file:

```config/locale_task_config.yaml```

in ManageIQ core.

Reply:
I think it's already updated:
```
 product/charts/miq_reports/*.yaml:
  - title
  - name
  - headers
```

Reply:
`db/fixtures/chargeback_rates.yml` is not in there

Reply:
I added this to `config/locale_task_config.yaml` and it didn't seem to do anything
```
db/fixtures/chargeback_rates.yml:
  - description
```

Reply:
Did you then run `bundle exec rake locale:update_all` to test the string extraction?

Reply:
I ran `bundle exec rake locale:update_all` and it added the strings to .po/.pot files but they don't have translations

Reply:
The fact that it extracted the string into `locale/manageiq.pot` is quite sufficient here. You don't have to worry about anything else really. What we need to do here is: 1. make sure the string are extracted (or at least that there's configuration to extract them) 2. apply gettext correctly

Reply:
Ok I'll make the PR for core